cognite/neat/_graph/extractors/_classic_cdf/_classic.py

```diff
@@ -109,6 +109,18 @@ def __init__(
         self._target_external_ids_by_type: dict[InstanceIdPrefix, set[str]] = defaultdict(set)
         self._labels: set[str] = set()
         self._data_set_ids: set[int] = set()
+        self._extracted_labels = False
+        self._extracted_data_sets = False
+
+    def _get_activity_names(self) -> list[str]:
+        activities = [core_node.extractor_cls.__name__ for core_node in self._classic_node_types] + [
```

Comment:
```suggestion
        activities = [dao.extractor_cls.__name__ for dao in self._classic_node_types] + [
```

Reply:
_ClassicCoreType is misleading name for class, and I am aware that I did review 3 months ago where it slipped my attention, where in reality it is sort of a Data Access Object class, with API endpoint + config how to create triples

Reply:
dao == Data Access Object